From 0eee3fb85dad084b6b42f7b219d8aa3a62620a27 Mon Sep 17 00:00:00 2001
From: Yi Zhao <yi.zhao@windriver.com>
Date: Wed, 28 Oct 2020 09:52:40 +0800
Subject: [PATCH] civetweb/CMakeLists.txt: do not search gcc-ar and gcc-ranlib

Do not search gcc-ar and gcc-ranlib. Specify GCC_AR and GCC_RANLIB in
EXTRA_OECMAKE to avoid host contamination.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 thirdparty/civetweb-1.10/CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/thirdparty/civetweb-1.10/CMakeLists.txt b/thirdparty/civetweb-1.10/CMakeLists.txt
index f559a04e..0689e3f6 100644
--- a/thirdparty/civetweb-1.10/CMakeLists.txt
+++ b/thirdparty/civetweb-1.10/CMakeLists.txt
@@ -246,11 +246,9 @@ set_property(CACHE CIVETWEB_CXX_STANDARD PROPERTY STRINGS auto c++14 c++11 c++98
 
 # Configure the linker
 if ("${CMAKE_C_COMPILER_ID}" STREQUAL "GNU")
-  find_program(GCC_AR gcc-ar)
   if (GCC_AR)
     set(CMAKE_AR ${GCC_AR})
   endif()
-  find_program(GCC_RANLIB gcc-ranlib)
   if (GCC_RANLIB)
     set(CMAKE_RANLIB ${GCC_RANLIB})
   endif()
-- 
2.17.1

